sympy/solvers/ode/tests/test_single.py

```diff
@@ -809,8 +822,8 @@ def _get_examples_ode_sol_factorable():
 
     'fact_11': {
         'eq': (f(x).diff(x, 2)-exp(f(x)))*(f(x).diff(x, 2)+exp(f(x))),
-        'sol': [], #currently dsolve doesn't return any solution for this example
-        'XFAIL': ['factorable']
+        'sol': [Eq(f(x), log(C1/(cos(C1*sqrt(-1/C1)*(C2 + x)) - 1))), Eq(f(x), log(C1/(cos(C1*sqrt(-1/C1)*(C2 - x)) - 1))), Eq(f(x), log(C1/(1 - cos(C1*sqrt(-1/C1)*(C2 + x))))), Eq(f(x), log(C1/(1 - cos(C1*sqrt(-1/C1)*(C2 - x)))))],
+        'dsolve_too_slow': True,
```

Comment:
Is this solution returned by dsolve with hint `Factorable`? If not, then move this example under this newly added solver examples and there keep `XFAIL` as key for Factorable as it was earlier.

Reply:
Yes, `Factorable` simplifies the equation and the method I have added gives one of the solutions to it. Previously, `dsolve` was unable to compute the solution, but now it can do it, so I have added the corresponding solution

Reply:
I think factorable factorises the ODE and then calls dsolve again. The new solver here is then able to return an answer for the factorised ODE which was previously not solvable.

Reply:
Yes

Reply:
Really `factorable` shouldn't be a "solver" hint. It should just be part of the preprocessing of `dsolve`. That's something that we can fix after `dsolve` is refactored.